package/yast2-storage-ng.spec

```diff
@@ -32,6 +32,8 @@ Requires:	libstorage-ng-ruby >= 4.1.89
 Requires:	rubygem(ruby-dbus)
 # Y2Packager::Repository
 Requires:	yast2-packager >= 3.3.7
+# NfsClient4partClient RefreshUI action
+Requires:	yast2-nfs-client >= 4.1.6
```

Comment:
As discussed in IRC, this require will produce a cyclic dependency since `yast-nfs-client` [already requires `yast-storage-ng`](https://github.com/yast/yast-nfs-client/blob/23eb048f5f52629d0e5fba2e7c9e773c0fa1d469/package/yast2-nfs-client.spec#L45).